src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,197 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+/**
+ * This class contains the task list and deals with add/delete list operations.
+ * @author Sim Jun Heng
+ * @version CS2103T AY21/22 Sem 2
+ */
+public class TaskList {
+    // ArrayList to store all your tasks
+    private ArrayList<Task> list;
+
+    // Constructor
+    public TaskList() {
+        list = new ArrayList<>();
+    }
+
+    /**
+     * Deletes task from the list.
+     *
+     * @param index the position of the task.
+     * @throws DukeException if position of the task exceeds what we have on the list.
+     */
+    public void deleteTask(int index) throws DukeException {
+        if (index >= list.size()) {
+            throw new DukeException("☹ OOPS!!! I'M SORRY, CAN'T FIND TASK");
+        } else {
+            Task task = list.remove(index);
+            System.out.println("NOTED. I'VE REMOVED THIS TASK:");
+            System.out.println(task);
+            System.out.println("NOW YOU HAVE " + list.size() + " TASKS IN THE LIST.");
+        }
+    }
+
+    /**
+     * Adds to do task into our current list.
+     *
+     * @param desc the task description entered by the user
```

Comment:
Something small that you have been doing consistently otherwise but have seemed to miss out here is the period after the parameter description in your javadocs. Very tiny detail but something good to be conscious about:)